service/handler.go

```diff
@@ -113,7 +113,7 @@ func EvaluateRequest(
 
 	permission := evaluatorSdk.Config()
 
-	rondInput, err := rondhttp.NewInput(req, env.ClientTypeHeader, mux.Vars(req), rondInputUser, nil)
+	rondInput, err := rondhttp.NewInput(&permission, req, env.ClientTypeHeader, mux.Vars(req), rondInputUser, nil)
```

Comment:
Why do we have a variable of type 'RondConfig' named 'permission'?  
Is this intentional?

Reply:
just a legacy naming